test/studies/shootout/reverse-complement/bradc/revcomp-blc.chpl

```diff
@@ -6,12 +6,15 @@
    and the Chapel #2 version by myself and Ben Harshbarger
 */
 
+use IO;
+
 param eol = "\n".toByte();      // end-of-line, as an integer
 
-const table = createTable();    // create the table of code complements
+param cmpl = b"          \n                                                 "
+            //      ABCDEFGHIJKLMNOPQRSTUVWXYZ      abcdefghijklmnopqrstuvwxyz
+           + b"     TVGH  CD  M KN   YSAABW R       TVGH  CD  M KN   YSAABW R";
```

Comment:
I trust you that this makes sense :)

Reply:
Yeah, a comment might be a good idea, but read this as a compact map from one ASCII character (an index from 0-127) to another (the byte stored in a given position), but a sparse map in which we only focus on the characters in the alphabet that will appear in fasta sequences along with newline.

But the cool part is that 12 SLOC got replaced by 2!